content/guide/avalanche9000-motivation.mdx

```diff
@@ -0,0 +1,94 @@
+
+
+### Current Economic Barriers
+
+![Staking Requirements](/common-images/multi-chain-architecture/acp-77/subnets-staking-requirements.png)
+The Primary Network currently has a staking requirement of 2,000 AVAX.
+
+If we assume a price of $20 per AVAX, this would be a $40,000 staking requirement per validator.
+
+To launch a Subnet today, validators must meet the staking requirement for each subsequent Subnet they choose to validate. This is typically a native token to the Subnet.
+This model can quickly lead to a high economic barrier of entry in creating new Subnets as validators must meet the 2,000 AVAX requirement in addition to the staking requirements of any Subnets they want to validate.
+
+### Lowering Economic Barriers
+![L1 Staking Requirements](/common-images/multi-chain-architecture/acp-77/l1s-staking-requirement.png)
+
+This upgrade will remove the requirement to stake 2,000 AVAX on the Primary Network, and instead only require validators to stake on L1 with their native token.
+This will lower the economic barrier to entry, and enable more validators to participate in the network.
+
+
+### Current Running Costs
+![Subnet Running Cost](/common-images/multi-chain-architecture/acp-77/subnet-running-cost.png)
+
+Currently, validators must validate the Primary Network in addition to each Subnet they are validating. This can lead to higher running costs for validators as they must maintain machines that can handle the load of both networks.
+
+### Lower Running Costs
+![L1 Running Cost](/common-images/multi-chain-architecture/acp-77/l1-running-cost.png)
+With ACP-77, validators will only need to validate the L1. This will lead to lower running costs for validators as they will not need to maintain machines that can handle the load of both the Subnets and Primary Network.
+
+### Current Compliance Barriers
+![Subnet Compliance](/common-images/multi-chain-architecture/acp-77/subnets-compliance.png)
+
+Many Institutions that want to adopt blockchain are struggling with exposure to public permissionless blockchains.
+The Primary Network is a public permissionless blockchain, and as such, it is not suitable to build on for many institutions.
+
+While institutions can launch their own Subnet as a private, proof of authority blockchain, they are still exposed to the Primary Network in some manner.
+
+This exposure can be circumvented by performing a partial sync of the Primary Network, but is an incomplete solution.
+### Better Institutional Compliance
+![L1 Compliance](/common-images/multi-chain-architecture/acp-77/l1s-compliance.png)
+ 
+Since validator sets of L1s are now distinct and seperate from the Primary Network, institutions can now launch their own L1s (be it public or private) without exposure to the public permissionless Primary Network.
```

Comment:
```suggestion
Since the validator sets of L1s will be distinct and separate from the Primary Network, institutions will be able to launch independent L1s (be it public or private) that are not required to participate in the Primary Network.
```